### DISCUSSION TITLE
Are SC's 1.4.4 Resize Text, 1.4.10 Reflow,  1.4.12 Text Spacing and 1.3.5 Identify Input Purpose applicable on Desktop App?

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#2861] [Q&A]
Hi Team,

I would like to know whether the Success Criterion's 1.4.4 Resize Text, 1.4.10 Reflow,  1.4.12 Text Spacing and 1.3.5 Identify Input Purpose applicable on Desktop App or not.

If yes, how we can test these Success Criterion's on Desktop App?

Thanks!!

**Comment by JAWS-test** [comment]
All SC of WCAG do not apply to desktop apps, as WCAG only applies to web content.

There is WCAG ITC for desktop apps, but it is outdated (based on WCAG 2.0) and not very relevant because it is not a recognized standard. https://www.w3.org/WAI/standards-guidelines/wcag/non-web-ict/

**Comment by JAWS-test** [comment]
What there are, however, are national rules that apply to desktop apps and refer to the WCAG, e.g. in the USA section 508 and in the countries of the EU the EN 301 549. If you look in these rules, you'll see that the SCs you mention also apply to desktop apps (insofar as the technology supports it.) 

I.e. SC 1.4.4 always applies (since I need to be able to adjust the font size in the OS or application OR or must comply with a minimum font size), whereas 1.4.12 only applies if I can adjust the text spacing in the application or OS.

**Comment by JAWS-test** [comment]
Example from EN 301 549:

11.1.4.4 Resize text
11.1.4.4.1 Resize text (open functionality)
Where ICT is non-web software that provides a user interface and that supports access to enlargement features of platform or assistive technology, it shall satisfy the WCAG 2.1 Success Criterion 1.4.4 Resize Text.
NOTE 1: Content for which there are software players, viewers or editors with a 200 percent zoom feature would automatically meet this success criterion when used with such players, unless the content will not work with zoom.
NOTE 2: This success criterion is about the ability to allow users to enlarge the text on screen at least up to 200 % without needing to use assistive technologies. This means that the application provides some means for enlarging the text 200 % (zoom or otherwise) without loss of content or functionality or that the application works with the platform features that meet this requirement.

11.1.4.4.2 Resize text (closed functionality)
Where ICT is non-web software that provides a user interface which is not able to access the enlargement features of platform or assistive technology, it shall meet requirement 5.1.4 (Functionality closed to text enlargement).
NOTE: Because the text rendering support in a closed environment may be more limited than the support found in user agents for the Web, meeting the present clause in a closed environment may place a much heavier burden on the content author.

11.1.4.12 Text spacing
Where ICT is non-web software that provides a user interface and that does not have a fixed size content layout area that is essential to the information being conveyed, it shall satisfy WCAG 2.1 Success Criterion 1.4.12 Text spacing.

**Comment by Myndex** [comment]
**US 508 rules** specify WCAG 2.0 A AA, and apply to government procurement, i.e. govt/govt funded, with exceptions for tech not commercially available, and for alternate faciliation.

There are other examples, such as the FAA reg that says 25% of airport kiosks must meet WCAG 2.

**From EN 301 549:**
 "..._suitable for use in public procurement¹ within Europe. The present document is intended to be used with web based technologies, non-web technologies and hybrids that use both_..."

EN 301 549 specifies WCAG 2.1 for many line items, even including direct links to the W3 document.

¹ _"public procurement" generally means government funded._
